src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+// Функція для конвертації зарплати в число
+function convertSalaryToNumber(salary) {
+  return +salary.replace(/[^0-9]/g, '');
+}
+
+function sortList(listProperty) {
+  const listArray = Array.from(listProperty);
+
+  // Сортуємо масив за зарплатою
```

Comment:
Remove comment. Try to write obvious code that explains all we need without comments)